src/components/radio/radio-group.tsx

```diff
@@ -28,10 +28,10 @@ const RadioGroup: FC<RadioGroupProps> = ({
   ...props
 }: RadioGroupProps) => {
   const handleChange = useCallback(
-    (value: string) => () => {
-      onChange(value);
+    (newValue: string) => () => {
```

Comment:
this is because the `value` name was shadowing `value` from outer scope
just a good measure